task_5/Toma.Lobanova/src/main/java/ua/od/hillel/DigitsSum/DigitsSum.java

```diff
@@ -0,0 +1,11 @@
+package ua.od.hillel.DigitsSum;
+
+public class DigitsSum {
+    public static void main(String... args){
+        long number=34567867890234L;
+        while(number/10L!=0){
```

Comment:
Looks a bit difficult to understand. Maybe it'd be better to make this problem without streams or move the stream into a separate method, so it is easier to get what's going on in the while loop.